### ISSUE TITLE
Newly tagged unpublished product shows as sold out/ missing product img in edit mode

### ISSUE THREAD

**Issue by nnnnat** [reactioncommerce/reaction#3673]
### Expected behavior
- Product should display with an "unpublished" notification
- Product should have correct product image

### Actual behavior
- Product is missing product image and listed as sold out

### Steps to reproduce the behavior
- As an Admin add a new tag to the tag nav
- Apply new tag to existing product
- Navigate to new tags product page

### Versions
```shell
Node: 9.4.0
NPM: 5.6.0
Meteor Node: 8.9.3
Meteor NPM: 5.5.1
Reaction CLI: 0.26.4
Reaction: 1.8.0
Reaction branch: release-1.8.0
```

**Comment by spencern** [comment]
@nnnnat I'd love to see a screencap or gif showing this issue.

Is this on the PDP page or on the product grid?

**Comment by nnnnat** [comment]
gif of the steps to reproduce

![unpublished-product-tag-nav](https://user-images.githubusercontent.com/1135948/36216018-4770ae4e-1173-11e8-9887-22ca6f993193.gif)

**Comment by aldeed** [comment]
@nnnnat This is fixed in 2.0.0 right?

**Comment by nnnnat** [comment]
@aldeed yes this is fixed in the new operator UI
